doc/nrf/releases/release-notes-1.9.0.rst

```diff
@@ -0,0 +1,881 @@
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Application development
+=======================
+
+* Updated:
+
+  * :ref:`app_build_system` by adding information about default enabling of experimental Kconfig option and the generation of warnings when experimental features are selected.
+  * :ref:`ug_edge_impulse` by adding instructions on how to download a model from a public Edge Impulse project.
+  * :ref:`ug_nrf_cloud` by adding a section that describes and compares MQTT and REST.
+
+* Removed:
+
+  * Using Pelion with the |NCS| user guide as part of removal of Pelion DM support.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+See `Samples`_ for lists of changes for the protocol-related samples.
+
+Matter
+------
+
+* Added:
+
+  *  ``EXPERIMENTAL`` select in Kconfig that informs that Matter support is experimental.
+
+* Updated:
+
+  * :ref:`ug_matter_configuring` by embedding the Matter tutorial video.
+  * :ref:`ug_matter_configuring_protocol` by adding information about setting Matter log levels and available device identification optional features. See sections :ref:`ug_matter_configuring_optional_log` and :ref:`ug_matter_configuring_device_identification` for details.
+  * :ref:`ug_matter_creating_accessory` by updating the described procedure to match the current Matter API.
+
+
+Thread
+------
+
+* Updated:
+
+  * Enabling of Thread 1.2. Refer to :ref:`thread_ug_thread_specification_options` and :ref:`thread_ug_feature_sets` for more information.
+  * :ref:`ug_thread_cert` by adding instructions for using VS Code extension as an alternative to command line when building the CLI sample for Thread certification using the Thread Test Harness.
+  * :ref:`thread_ug_feature_sets` by adding support for SRP client and removing rarely used features.
+
+Zigbee
+------
+
+* Added:
+
+  * Experimental support for Zigbee Cluster Library ver8 (ZCL8), included in :ref:`nrfxlib:zboss` v3.11.1.177.
+  * Support for Zigbee Base Device Behavior v3.0.1 (BDB 3.0.1).
+  * Set of Zigbee libraries with binary trace logs enabled.
+  * Stability and performance improvements.
+  * Documentation for collecting Zigbee trace logs. See :ref:`ug_zigbee_configuring_zboss_traces` for more information.
+
+
+* Updated:
+
+  * Support for nRF5340 and for the combination of nRF5340 and nRF21540. The support is not experimental anymore.
+  * :ref:`nrfxlib:zboss` to v3.11.1.0 and platform v5.1.1 (``v3.11.1.0+v5.1.1``).
+  * :ref:`ZBOSS Network Co-processor Host <ug_zigbee_tools_ncp_host>` package to the new version v2.1.1.
+  * :ref:`lib_zigbee_osif` library with Kconfig options that allow to either reset or halt the device upon ZBOSS stack assert. Reset is enabled by default.
+
+* Fixed:
+
+  * Several bugs.
+
+HomeKit
+-------
+
+* Added:
+
+  * Nordic proprietary Device Firmware Update over Bluetooth LE (Nordic DFU) based on mcumgr.
```

Comment:
```suggestion
  * Nordic proprietary Device Firmware Update over Bluetooth LE (Nordic DFU) based on mcumgr.
    Before using the new DFU solution, make sure to check HomeKit known issues for this release.
```
@umapraseeda We will be a adding a known issue for this and it would be nice to ref to it from here.
@maciejpietras  , could you provide text for the new known issue?

Reply:
We will update known issue list until tomorrow.

Reply:
Added this statement.

Reply:
Removing this statement as suggested by @maciejpietras .